Client/Application/AppDelegate.swift

```diff
@@ -257,9 +257,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             log.warning("Cannot handle \(components.scheme) URL scheme")
             return false
         }
+        
+        // Extract optional FxA deep-linking options
+        let fxaQuery = url.getQuery()
+        let fxaParams: FxALaunchParams
+        fxaParams = FxALaunchParams(view: fxaQuery["fxa"], email: fxaQuery["email"], access_code: fxaQuery["access_code"])
```

Comment:
Yea it should be fine 👍 You've declared the launch params as optional so it shouldn't crash. They are allowed to be nil